flask-server/main.py

```diff
@@ -114,11 +116,46 @@ def decode_auth_token(auth_token):
         return 'Signature expired. Please log in again.'
     except jwt.InvalidTokenError:
         return 'Invalid token. Please log in again.'
+
+@app.route('/apis/add-clink', methods=['POST'])
+def add_clink():
+    try:
```

Comment:
Could remove try, except statement for consistency for code(sign-up api has the same functionality but w/o try, except statement). I originally added it to provide some kind of error response to test redux store dispatch actions. Realistically there won't be an exception with storing and fetching a clink.

Reply:
done!